content/en/docs/security/config-best-practices.md

```diff
@@ -0,0 +1,155 @@
+
+<!--- TODO: SHOULD configure obfuscation/scrubbing of sensitive metadata. How? Give more details and/or link to an existing document -->
+
+Use OpenTelemetry Collector's `redaction` processor to scrub sensitive data.
+
+<!--- TODO: Give example config for the redaction processor or remove this line. --->
+
+### Safeguarding resource utilization
+
+Processors also offer safeguards for resource utilization. After implementing
+safeguards for resource utilization in your
+[hosting infrastructure](/security/hosting-best-practices/), make sure your
+OpenTelemetry Collector configuration uses these safeguards.
+
+<!-- start same page content in hosting-best-practices -->
+
+The `batch` and `memory_limiter` processors help ensure the OpenTelemetry
+Collector is resource efficient and does not run out of memory when overloaded.
+These two processors should be enabled on every defined pipeline.
+
+For more information on recommended processors and how to order them in your
+configuration, see the
+[Collector processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor)
+documentation.
+
+<!-- /end same page content in hosting-best-practices -->
+
+## Extensions
+
+While receivers, processors, and exporters handle telemetry directly, extensions
+serve different needs.
+
+<!--- TODO: Extensions SHOULD NOT expose sensitive health or telemetry data. How? What can you do? -->
+
+### Health and telemetry
+
+Extensions are available for health check information, Collector metrics and
+traces, and generating and collecting profiling data. When enabled with their
+default settings, all of these extensions, except the health check extension,
+are accessible only locally to the OpenTelemetry Collector. Take care to protect
+sensitive information when configuring these extensions for remote access, as
+they might expose it accidentally.
+
+### Forwarding
+
+A forwarding extension is used when you need to collect telemetry that's not
+natively supported by the Collector. For example, the `http_forwarder` extension
+can receive and forward HTTP payloads. Forwarding extensions are similar to
+receivers and exporters so the same security considerations apply.
+
+### Collector's internal telemetry
+
+<!--- INSERT RECOMMENDATIONS HERE. For example:
+
+1. Remove zPages.
+1. Remove configuration endpoints.
+-->
+
+### Observers
+
+An observer is a component that discovers services in endpoints. Other
+components of the OpenTelemetry Collector, such as receivers, can subscribe to
+these extensions to be notified of endpoints coming or going.
+
+Observers might require certain permissions in order to discover services. For
+example, the `k8s_observer` requires certain RBAC permissions in Kubernetes,
+while the `host_observer` requires the OpenTelemetry Collector to run in
+privileged mode.
+
+<!--- But what about Juraci's comment here: https://github.com/open-telemetry/opentelemetry.io/pull/3652/files?diff=unified&w=0#r1417409370 --->
```

Comment:
It's OK -- what we shouldn't be telling people is to run things as root or blindly disable security protections (like selinux).